### PR TITLE
Rearrange network interfaces page

### PR DESCRIPTION
**Pull request by Davo36**
Added IP addresses and netmask to the network-interfaces page.

### CODE REVIEW

File: management-interface.go [hunk 1]
```diff
@@ -141,6 +141,65 @@ func IndexHandler(w http.ResponseWriter, r *http.Request) {
...
+	return IPAdresses
+}
+
+// Get the IP address and net mask for each interface.
+func getInterfaceProperties(interfaces []net.Interface) []interfaceDetails {
+
+	var ifacesDetails []interfaceDetails
+
+	for _, iface := range interfaces {
+		netmask := getNetmask(iface.Name)
```
Comment (mjs) [inline management-interface.go]: You don't need this at all. There isn't one netmask per interface, it's per address, and you're already getting it when you get the IP addresses. The netmask is represented by the "/24" (or other number) that's already being printed at the end of each address.

The separate retrieval and output of netmasks can be removed.

FYI: "/24" is equivalent to "255.255.255.0". See also: https://www.iplocation.net/subnet-mask

Comment (Davo36) [reply management-interface.go]: Ok, thanks Menno, I will fix it.

File: management-interface.go [hunk 2]
```diff
@@ -141,41 +141,14 @@ func IndexHandler(w http.ResponseWriter, r *http.Request) {
...
-			netmaskStr = rows[1][netmaskPos+8 : broadcastPos]
-		}
-	}
-	return netmaskStr
-}
-
 // Get the IP address for a given interface.  There can be 0, 1 or 2 (e.g. IPv4 and IPv6)
 func getIPAddresses(iface net.Interface) []string {
 
 	var IPAdresses []string
```
Comment (mjs) [inline management-interface.go]: Missing a "d" here

File: management-interface.go [hunk 3]
```diff
@@ -184,43 +157,33 @@ func getIPAddresses(iface net.Interface) []string {
...
 
-// Get the IP address and net mask for each interface.
-func getInterfaceProperties(interfaces []net.Interface) []interfaceDetails {
-
-	var ifacesDetails []interfaceDetails
-
-	for _, iface := range interfaces {
-		netmask := getNetmask(iface.Name)
-		addresses := getIPAddresses(iface)
+// NetworkInterfacesHandler - Show the status of each newtwork interface
```
Comment (mjs) [inline management-interface.go]: network

File: html/network-interfaces.html [hunk 4]
```diff
@@ -24,12 +24,10 @@ <h2>Network Interfaces<br></h2>
                         <div class="card-body">
                             <h5 class="card-title"><strong>{{.Name}}</strong></h5>
                             <h6>IP Address(s):</h6>
```
Comment (mjs) [inline html/network-interfaces.html]: (es)

**Comment by Davo36** [comment]
Closes #416

**Pull request merged**